Docs/Notes/Readme.backtrace

```diff
@@ -19,27 +19,23 @@ with TEST=TRUE or DEBUG=TRUE in C++ AMReX, or TEST=t or NDEBUG= in
 Fortran AMReX.  In C++ AMReX, one can also control the setting using
 the runtime parameter, fab.init_snan.
 
-If it is compiled with BACKTRACE=TRUE, one can get more information
-than the backtrace of the call stack by instrumenting the code.  (This
-is in C++ code only.)  Here is an example.  You know the line "Real
-rho = state(cell,0);" is causing a segfault.  You could add a print
-statement before that.  But it might print out thousands (or even
-millions) of line before it hits the segfault.  With BACKTRACE, you
-could do
+One can get more information than the backtrace of the call stack by
```

Comment:
Looks good! Can we either link this file from the `.rst` manual or convert it to `.rst` and include it? This is good to know but I could not find it initially without a hint :)

Reply:
OK.  I will move it to the manual.